server/httpin/service.go

```diff
@@ -20,56 +20,28 @@ type Service interface {
 }
 
 type service struct {
-	router *mux.Router
+	rh *Handler
 }
 
 var _ Service = (*service)(nil)
 
-func NewService(router *mux.Router, conf config.Service, proxy proxy.Service, appServices appservices.Service,
-	initf ...func(*mux.Router, config.Service, proxy.Service, appservices.Service)) Service {
+func NewService(mm *pluginapi.Client, config config.Service, log utils.Logger, session incoming.SessionService, router *mux.Router, proxy proxy.Service, appServices appservices.Service,
```

Comment:
0/5 Why not router first? I'd go `router, mm, log, config, services, ...funcs`

Reply:
35203a7